### DISCUSSION TITLE
Scikit Learn Termux Package

### DISCUSSION THREAD

**Discussion by Ystyphn** [termux/termux-packages#18196] [Q&A] [answered]
Hello, I just want to ask if there are scikit-learn package for termux that could be installed via pkg? Scikit learn couldn't be installed even with `pip install scikit-learn --no-build-isolation`

![Screenshot_20231008-222236](https://github.com/termux/termux-packages/assets/83481828/12aef700-27f3-430c-bbc9-743aae44d1ba)

I already installed scipy, pandas, numpy and matplotlib.

**Comment by sylirre** [comment, marked as answer]
You need to install `wheel` python module to solve the error about bdist_wheel.

**Reply by Ystyphn** [reply]
It's taking too long, it has been more than 30 mins now

![Screenshot_20231009-084545](https://github.com/termux/termux-packages/assets/83481828/4b49f383-28ff-4572-8076-2a04747cb1cc)

**Reply by ddev67440-sketch** [reply]
Same

**Reply by sylirre** [reply]
@ddev67440-sketch May take more than an hour. Modules that use native extensions will compile them on device because Termux is a non-conventional environment (https://wiki.termux.dev/wiki/Differences_from_Linux).

Use `proot-distro` (Debian / Ubuntu) if you specifically prefer everything to be effortless.

**Comment by Ystyphn** [comment]
I already found the answer. It just takes too long but it works!

**Comment by ddev67440-sketch** [comment]
![Uploading Screenshot_2025_1224_122014.jpg…]()
